superchain/superchain.go

```diff
@@ -18,6 +21,12 @@ var superchainFS embed.FS
 //go:embed extra/addresses extra/bytecodes extra/genesis extra/genesis-system-configs
 var extraFS embed.FS
```

Comment:
I think we can also update this since go embeds subtree files recursively in directories (see below).
```suggestion
//go:embed extra
var extraFS embed.FS
```

Reply:
I wanted to skip the `README.md` in this case

Reply:
Ah that makes sense, good call